source/renderer/app/api/ada/index.js

```diff
@@ -689,12 +702,15 @@ export default class AdaApi {
     Logger.debug('AdaApi::nextUpdate called');
     let nextUpdate = null;
     try {
-      // TODO: add flow type definitions for nextUpdate response
-      const response: Promise<any> = await nextAdaUpdate({ ca });
+      const response = await nextAdaUpdate({ ca });
+      const { projectVersion, softwareInfo } = response;
+      const { version, applicationName } = softwareInfo;
+
       Logger.debug('AdaApi::nextUpdate success: ' + stringifyData(response));
-      if (response && response.cuiSoftwareVersion) {
+      if (version > projectVersion) {
```

Comment:
@nikolaglumac 

This PR is ready to be merged in terms of review approvals but I need a second opinion on intended functionality here before I pull the trigger. This is the `nextAdaUpdate` API method which will determine if the node update modal is shown or not. Previously, the update modal was shown `if (response && response.cuiSoftwareVersion)`, but the V1 data looks totally different as you can see here on lines `705 - 714`. I am not really sure if the check I have implemented is correct and it's difficult to test because I am not sure about the details of what I should be looking for in the response. For example, I don't know what `version` and `projectVersion` are actually referencing. 🥇 :trollface:

Reply:
@MarcusHurney this is something which I have mentioned on our meeting yesterday. There will be a specific endpoint for this purpose.

Reply:
@nikolaglumac 
My brain must have not been fully awake yet, but glad to know it now. This is ready to go then.